spec/associations/active_record_extensions_spec.rb

```diff
@@ -166,6 +166,12 @@ class Book < ActiveRecord::Base
           expect(school.country).to eq(nil)
         end
 
+        it "doesn't interfere with AR's belongs_to arguments" do
+          allow(ActiveRecord::Base).to receive(:belongs_to).and_call_original
+          School.belongs_to :country
+          expect(ActiveRecord::Base).to have_received(:belongs_to).with(:country, nil)
+        end
+
```

Comment:
[FYI]
Before the fix, the rspec case failed as follows.
```ruby
$ bundle exec rspec spec/associations/active_record_extensions_spec.rb:149
Run options: include {:locations=>{"./spec/associations/active_record_extensions_spec.rb"=>[149]}}
.F....

Failures:

  1) ActiveHash::Base active record extensions ActiveHash::Associations::ActiveRecordExtensions#belongs_to doesn't interfere with AR's belongs_to arguments
     Failure/Error: expect(ActiveRecord::Base).to have_received(:belongs_to).with(:country, nil)
     
       #<ActiveRecord::Base (class)> received :belongs_to with unexpected arguments
         expected: (:country, nil)
              got: (:country, nil, {:class_name=>"Country"})
       Diff:
       @@ -1 +1 @@
       -[:country, nil]
       +[:country, nil, {:class_name=>"Country"}]
       
     # ./spec/associations/active_record_extensions_spec.rb:172:in `block (4 levels) in <top (required)>'

Finished in 0.17329 seconds (files took 0.32334 seconds to load)
6 examples, 1 failure

Failed examples:

rspec ./spec/associations/active_record_extensions_spec.rb:169 # ActiveHash::Base active record extensions ActiveHash::Associations::ActiveRecordExtensions#belongs_to doesn't interfere with AR's belongs_to arguments
```

Reply:
adding in the parameter `any` is your friend

Reply:
@kbrock 
Thank you for your advice! 
I will work on, please wait for a while.